packages/headless/src/composables/comboBox/createComboBox.ts

```diff
@@ -1,50 +1,85 @@
-import { computed, ref, type Ref } from "vue";
+import { computed, ref, type MaybeRef, type Ref } from "vue";
 import { createBuilder } from "../../utils/builder";
 import { createId } from "../../utils/id";
+import { createListbox } from "../listbox/createListbox";
+
+export type CreateComboboxOptions<TValue extends string> = {
+  /**
+   * Labels the listbox which displays the available options. E.g. the list label could be "Countries" for a combobox which is labelled "Country".
+   */
+  listLabel: MaybeRef<string>;
```

Comment:
Is a listbox label needed when the combox already has a label and controls the listbox? As I understand no label is needed then for the listbox itself

Reply:
The listbox label describes the list items e.g., "Countries" and the combobox label describes the input context e.g., "Your country of residence"

Reply:
In the [listbox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/), it says the listbox label is not required if used inside other control components like a comboox
![image](https://github.com/SchwarzIT/onyx/assets/67898185/c583f5cc-c245-47db-83ea-ce30bbcb19f1)

Reply:
Will change that in the follow-up PR :)